sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SpanDataMapper.java

```diff
@@ -815,13 +813,15 @@ private TelemetryItem createExceptionTelemetryItem(String errorStack, SpanData s
         } else {
             setOperationName(telemetryBuilder, span.getAttributes());
         }
-        setTime(telemetryBuilder, span.getEndEpochNanos());
+        setTime(telemetryBuilder, event.getEpochNanos());
         setSampleRate(telemetryBuilder, sampleRate);
 
+        // should this map the span attributes or the event attributes?
         MAPPINGS.map(span.getAttributes(), telemetryBuilder);
 
         // set exception-specific properties
-        setExceptions(errorStack, span.getAttributes(), telemetryBuilder);
+        String errorStack = event.getAttributes().get(SemanticAttributes.EXCEPTION_STACKTRACE);
+        setExceptions(errorStack, event.getAttributes(), telemetryBuilder);
```

Comment:
this was the problem here, it was passing in span attributes instead of event attributes

prior to #41397, type name would be parsed from the stack trace instead of read from the attributes passed in here